packages/generator/ni_measurement_plugin_sdk_generator/client/__init__.py

```diff
@@ -49,7 +48,7 @@ def _create_file(
 
 
 @click.command()
-@click.argument("measurement_service_class")
+@click.argument("measurement_service_class", required=False, nargs=-1)
```

Comment:
`required=False` is the default. Consider removing per the Click documentation:
https://click.palletsprojects.com/en/8.1.x/arguments/#variadic-arguments

> we believe scripts should gracefully degrade into becoming noops if a variadic argument is empty